CMakeLists.txt

```diff
@@ -113,14 +113,14 @@ target_compile_features (sophus INTERFACE
 add_custom_target(other SOURCES ${SOPHUS_OTHER_FILES})
 
 # Create 'test' make target using ctest
-option(BUILD_TESTS "Build tests." ON)
+option(BUILD_TESTS "Build tests." OFF)
```

Comment:
Changing the defaults here seems like it could be breaking for others.  Suggest this is just done at the configuration level.